build/pluginctl/main.go

```diff
@@ -85,6 +99,23 @@ func getClient() (*model.Client4, error) {
 	return nil, errors.New("one of MM_ADMIN_TOKEN or MM_ADMIN_USERNAME/MM_ADMIN_PASSWORD must be defined")
 }
 
+func getUnixClient(socketPath string) (*model.Client4, error) {
+	_, err := net.Dial("unix", socketPath)
+	if err != nil {
+		return nil, err
+	}
+
+	tr := &http.Transport{
+		Dial: func(network, addr string) (net.Conn, error) {
+			return net.Dial("unix", socketPath)
+		},
+	}
+	client := model.NewAPIv4Client("http://_")
+	client.HttpClient = &http.Client{Transport: tr}
+
+	return client, nil
```

Comment:
We can use [the recently merged `NewAPIv4SocketClient` function here](https://github.com/mattermost/mattermost-server/blob/b372b98aca865b037cadcc9b3c53f19f7ed63dc3/model/client4.go#L122):

```suggestion
	return model.NewAPIv4SocketClient(socketPath), nil
```

Reply:
@mgdelacroix Same comment about this change. I'd like to get @lieut-data's opinion on this.